docs/en/quick-answers/troubleshooting/turn-debug-logging-on.md

```diff
@@ -163,3 +163,7 @@ If you’re running the tank command and want to see debug output in your termin
     You are currently running version v0.15.18 of the Shotgun Pipeline Toolkit
     No need to update the Toolkit Core API at this time!
     DEBUG [10:11:39 981.74405098]: Exiting with exit code None
+
+## Examining debug outputs in log files
```

Comment:
I can see why you've moved this down here, I agree I don't think it was in the right place before.
Just a suggestion and feel free to ignore, but I think probably 90% of people are just going to go with the Desktop checkbox, and probably not read the whole of what is quite a quite lengthy article.

Maybe we could move this up to just below the Desktop checkbox section?
And then sub group the rest of the methods as other options or somthing.

Or maybe create a TOC at the top of the doc?

Reply:
I'll move it to the top, and reword it to be a quick preface to the rest of the content

Reply:
Moved it up to the very top as a note. This was a good moment to throw in an intro sentence for the whole doc, too. Good call!